cluster/gce/win1803/install-ssh.psm1

```diff
@@ -17,31 +17,46 @@
   Library for installing and running Win32-OpenSSH.
 #>
 
+# Starts the Win32-OpenSSH services and configures them to automatically start
+# on subsequent boots.
+function Start_OpenSshServices {
+  ForEach ($service in ("sshd", "ssh-agent")) {
```

Comment:
Do we expect the list of services or the ops to start them to grow over time? Otherwise a for loop might be overkill here.

Reply:
No, we don't expect it to grow. I'd prefer to leave it as-is but it's not a strong preference.